tools/clang/test/CodeGenSPIRV/passthru-ps.hlsl2spv

```diff
@@ -6,7 +6,8 @@ float4 main(float4 input: COLOR): SV_TARGET
 
 
 // TODO:
-// OpEntryPoint Fragment %main "main"
+// OpEntryPoint Fragment %main "main" %f4_inputVar %f4_outputVar
```

Comment:
We should also consider checking in a `passthru-vs`. Not necessary converting all the contents in it, but at least check that the entry point is generated correctly.